docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -78,49 +78,109 @@ Pick one of the following plugins:
 
 - [babel-plugin-import](https://github.com/ant-design/babel-plugin-import) with the following configuration:
 
+  `yarn add -D babel-plugin-import`
+
+  Create a `.babelrc.js` file in the root directory of your project:
+
   ```js
-  plugins: [
+  const plugins = [
     [
-      'babel-plugin-import',
+      "import",
       {
-        libraryName: '@material-ui/core',
-        libraryDirectory: 'esm', // or '' if your bundler does not support ES modules
-        camel2DashComponentName: false,
+        "libraryName": "@material-ui/core",
+        // Use '"libraryDirectory": "",' if your bundler does not support ES modules
+        "libraryDirectory": "esm",
+        "camel2DashComponentName": false
       },
-      'core',
+      "core"
     ],
     [
-      'babel-plugin-import',
+      "import",
       {
-        libraryName: '@material-ui/icons',
-        libraryDirectory: 'esm', // or '' if your bundler does not support ES modules
-        camel2DashComponentName: false,
+        "libraryName": "@material-ui/icons",
+        // Use '"libraryDirectory": "",' if your bundler does not support ES modules
+        "libraryDirectory": "esm",
+        "camel2DashComponentName": false
       },
-      'icons',
-    ],
-  ],
+      "icons"
```

Comment:
Single quotes?

Reply:
This is a little too pedantic imho. I'm following the plugin's directions.

Reply:
I don't think that we should consider the convention of the plugin. I think that we should look at what prettier does by default.